.php_cs.dist

```diff
@@ -51,6 +51,7 @@ return PhpCsFixer\Config::create()
         'nullable_type_declaration_for_default_null_value' => [
             'use_nullable_type_declaration' => false,
         ],
+        'mb_str_functions' => true,
```

Comment:
Not all string functions should be migrated, make sure this is removed before merge!

Reply:
Why not all?

Reply:
Why all?

We can migrate all text functions as long as:
- it makes sense (like `strpos('ascii char')`, `strtoupper($http_method)`, strtoupper('hex string') are great examples where is does not)
- if the performance is not critical (like for magic name getters)
...

When never:
- if the string is binary - like checking lenght of binary file (ex. png image)
- when the lenght is checked for some storage limitation - a lot of applications limit the bytes count instead of Unicode symbols/chars.

Reply:
OK I rolled back part of changes in other repos too. Please review them when you have time.